src/app/badges/achievements-list/achievements-list.component.html

```diff
@@ -0,0 +1,27 @@
+<h2 class="title">full list of achievements</h2>
+
+<table mat-table [dataSource]="achievements">
+  <ng-container matColumnDef="title">
+    <th mat-header-cell *matHeaderCellDef> Achievement </th>
+    <td mat-cell *matCellDef="let element">
+      <div class="achievement-info">
+        <img src="{{ element.image }}" alt="achieve image" class="achievement__image">
```

Comment:
css classes should be written in .kebab-case. please change achievement__image to achievement-image